src/discord/cache.js

```diff
@@ -60,8 +60,9 @@ class DiscordCache {
 
   #load () {
     const rawFile = fs.readFileSync(cacheFilePath)
-    const messages = JSON.parse(rawFile).messages
-    const channels = JSON.parse(rawFile).caches
+    const json = JSON.parse(rawFile);
```

Comment:
```suggestion
    const json = JSON.parse(rawFile)
```

bad semicolon, nobody likes you